src/collective/elasticsearch/tests/test_catalog.py

```diff
@@ -45,15 +55,36 @@ def test_delete_object(self):
             title='Some Event')
         obj_uid = getUID(obj)
         self.portal.manage_delObjects(['event_to_delete'])
+        processQueue()
         self.assertEqual(self.get_hook().index, {})
         self.assertEqual(self.get_hook().remove, {obj_uid})
 
+    def test_moved_content(self):
+        """ content moved by content rules should remove the original catalog
+            entry
+        """
+        target = api.content.create(container=self.portal, type='Folder',
+                                    id='target')
+        source = api.content.create(container=self.portal, type='Folder',
+                                    id='source')
+        e = MoveAction()
+        e.target_folder = '/target'
+
+        obj = api.content.create(container=source, type='Document', id='doc')
+        ex = getMultiAdapter((target, e, DummyEvent(obj)), IExecutable)
+        self.assertEqual(True, ex())
+        catalog = api.portal.get_tool('portal_catalog')
+        self.assertEqual(len(catalog(portal_type='Document',
+                                     path='/plone/source')), 0)
+        self.assertEqual(len(catalog(portal_type='Document',
+                                     path='/plone/target')), 1)
+
 
 if HAS_ATCONTENTTYPES:
-    from collective.elasticsearch.testing import ElasticSearch_FUNCTIONAL_TESTING_AT  # noqa
+    from collective.elasticsearch.testing import \
+        ElasticSearch_FUNCTIONAL_TESTING_AT  # noqa
```

Comment:
Please keep imports on one line.

Reply:
I believe this one was failing flake8 because of line length, I'll double check

Reply:
Yeah without the line break this is 93 chars long and failing flake8 test. I don't know how you want to resolve both requirements - change it into a relative import?

Reply:
The noqua comment tells flake8 to ignore errors on this line. See that in [master](https://github.com/collective/collective.elasticsearch/blob/9f5d42e031997069ae414dd4b12edab93e0ccaaa/src/collective/elasticsearch/tests/test_search.py#L213) it's in a line and the build is green.

Reply:
Not sure what my issue was earlier then, corrected this and it passed